db/migrate/20230502232411_add_user_and_post_ref_to_comment.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserAndPostRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, references: :posts, foreign_key: true
+    add_reference :comments, :user, references: :users, foreign_key: true
```

Comment:
- [ ] @sajibarafatsiddiqui great work on your migration files, however, we should reference users using athour_id instead of the current user_id, this will be in line with the ERD where all relationships with the user are done using the author_id
![image](https://user-images.githubusercontent.com/61408860/236018983-d6b805b4-411a-42cb-8002-689273723b80.png)

Reply:
changed it accordingly